src/commands/graduate.ts

```diff
@@ -0,0 +1,106 @@
+import {
+    ChatInputCommandInteraction,
+    GuildMemberRoleManager,
+    SlashCommandBuilder,
+} from 'discord.js';
+
+/**
+ * Gives 'graduate' role to the user that already has role 'verified'.
+ */
+
+export const data = new SlashCommandBuilder()
+    .setName('graduate')
+    .setDescription('Identifies user permanently as graduate.')
+    .addStringOption((option) =>
+        option
+            .setName('confirmation')
+            .setDescription(
+                "Write 'confirm' if you want to set your status to graduate."
```

Comment:
Just a small nit - is this really necessary? Is this just supposed to make sure that you do not give yourself role accidentally or is there some other reason?

Reply:
Hi, it is exactly as you mentioned. Just an additional step to make sure that you do not give yourself role accidentally.